internal/tests/cli/boundary/target.bats

```diff
@@ -69,7 +70,7 @@ load _helpers
 
 @test "boundary/target: admin user can add default host set to created target" {
   local id=$(target_id_from_name $DEFAULT_P_ID $TGT_NAME)
-  run assoc_host_sources $id $DEFAULT_HOST_SET
+  run add_target_host_sources $id $DEFAULT_HOST_SET
   echo "$output"
   [ "$status" -eq 0 ]
 }
```

Comment:
The test below this one uses `target_has_host_source_id`. I'm a little surprised that it's working given the argument changes made to `target_has_host_source_id`. What's going on there?

Reply:
`target_has_host_source_id ` does not return 1 by default if it does not match any of the conditions so somehow it was just passing. I'll update it to return 1 and add a format here as well